std/process.d

```diff
@@ -1474,7 +1474,10 @@ unittest // tryWait() and kill()
         TestScript prog = "while true; do sleep 1; done";
     }
     auto pid = spawnProcess(prog.path);
-    Thread.sleep(dur!"seconds"(1));
+    version (CRuntime_Bionic)
+        Thread.sleep(dur!"msecs"(5));
+    else
+        Thread.sleep(dur!"seconds"(1));
```

Comment:
Sleeping processes launched from the command-line seem to be killed very quickly on Android, within 5-10 ms in my experience. Since the point here is to check the signal after we kill it, shorten the sleep time to make sure the process is still alive when we kill it.